Dockerfile

```diff
@@ -32,6 +32,9 @@ WORKDIR /src/semgrep
 COPY . .
 
 # remove files and folders that aren't necessary for the semgrep-core build
+# coupling: see the (dirs ...) directive in the toplevel dune file for the list
+# of directories containing OCaml code and which should not be added below
+# (except js/ which contains OCaml code but is not used to build semgrep-core)
```

Comment:
This idea is probably too complex, but I wonder if we could extract those dirs from the dune file to ensure that we're including the necessary folders?